### ISSUE TITLE
feed2all.org: nuisance, ads and other issues

### ISSUE THREAD

**Issue by SirCoda** [uBlockOrigin/uAssets#26479]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.). Comment in the old issue threads even when they are closed or even if you have a different problem.
- [X] I did not remove any of the [default filter lists](https://github.com/user-attachments/assets/1879fc88-09bc-4de4-9e2f-3834a6ab396a), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://feed2all.org/football/feed-los-angeles-chargers-tampa-bay-buccaneers-2all5gmeh?l=1221001955`

### Category

nuisance

### Description

Site has some kind of layering that prevents blocking some elements.  Even when I turn off blocking or use a fresh browser with only uBlock active, blocking is detected and prevented.  I would prefer to use firstrowsports.to but they have completely stopped ublock from working there.  Feed2all.org at least still uses their old methods of thwarting uBlock.  Sometimes I can get around their attempts to stop me from watching my games, sometimes not.

### Other extensions used

ClearURLs addon.  Tiletabs WE, Google opt-out for analytics.

For some reason this page will not let me drag and drop a .jpg screenshot in the section below this one.

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.61.2
Firefox: 133
filterset (summary):
 network: 141751
 cosmetic: 24
 scriptlet: 21452
 html: 2226
listset (total-discarded, last-updated):
 default:
  user-filters: 96-42, never
  ublock-filters: 41175-11491, 15m
  ublock-badware: 12466-702, 15m
  ublock-privacy: 1640-35, 15m
  ublock-unbreak: 2600-340, 15m
  ublock-quick-fixes: 197-41, 15m
  easylist: 78570-37557, 15m
  easyprivacy: 53308-76, 15m
  urlhaus-1: 23270-0, 15m
  plowe-0: 3549-992, 15m
filterset (user): [array of 98 redacted]
trustedset:
 added: [array of 11 redacted]
switchRuleset:
 added: [array of 3 redacted]
userSettings:
 collapseBlocked: false
 parseAllABPHideFilters: false
 showIconBadge: false
 userFiltersTrusted: true
hiddenSettings: [none]
supportStats:
 allReadyAfter: 162 ms (selfie)
 maxAssetCacheWait: 53 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 24
 network:
  blockestvs.xyz: 2
  d13k7prax1yi04.cloudfront.net: 1
  google-analytics.com: 1
  googletagmanager.com: 8
  jsdelivr.net: 3
  partyroll.xyz: 6
  vawcdhhgnqkrif.com: 1
  visariomedia.com: 1
  yihjrdibdpy.com: 1
 extended:
  ##.footer
  ##.right
  ##.show.cover
  ###obrazek
```
</details>

**Comment by SirCoda** [comment]
![Blocked](https://github.com/user-attachments/assets/6ac2f0d5-5d6e-4814-87dd-63d9f2313dcf)

**Comment by mapx-** [comment]
@SirCoda no match available, test the site with the last filters added in the commit above

**Comment by SirCoda** [comment]
Still doesn’t work. Sent from Mail for Windows From: D4niloMRSent: Tuesday, December 17, 2024 2:23 PMTo: uBlockOrigin/uAssetsCc: SirCoda; MentionSubject: Re: [uBlockOrigin/uAssets] feed2all.org: nuisance, ads and other issues (Issue #26479) Closed #26479 as completed.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>
